rest_framework/serializers.py

```diff
@@ -773,7 +773,7 @@ def errors(self):
 # ModelSerializer & HyperlinkedModelSerializer
 # --------------------------------------------
 
-def raise_errors_on_nested_writes(method_name, serializer, validated_data):
+def raise_errors_on_nested_writes(method_name, serializer, validated_data, model_field_info):
```

Comment:
Is there any reason to pass in the model field info instead of getting it from the serializer class here? Getting from the serializer would mean the signature doesn't need to change.